chapters/001-getting-started/README.md

```diff
@@ -313,63 +258,37 @@ From `boudi`, we see some ping output like this:
 64 bytes from 10.1.1.3: icmp_seq=3 ttl=64 time=0.240 ms
 ```
 
-Basically, all you need to know about this is that "ping" is a program that
-sends packets across the network using a protocol called ICMP (which stands for
-Internet Control Message Protocol): "echo request" and "echo reply" are two
-types of ICMP message [read more about them here](https://docs.netapp.com/us-en/e-series-santricity/sm-hardware/what-are-icmp-ping-responses.html).
-What we see here in this ping message is that it has both sent a packet to the
-destination, and the destination has replied. The `icmp_seq=3` designation marks
-each individual request/response pair. If the ping did not go through, you might
-see various error messages, but the most common is that the `ping` command
-replies with `Request timeout for icmp_seq 0` type messages.
-
-On the container that is being pinged (pippin), we see quite a bit more
-information from tcpdump. First, we see a series of messages that have `ARP` in
-them. ARP is a protocol that you can learn all about on [the internet](https://www.fortinet.com/resources/cyberglossary/what-is-arp).
-Basically, it allows a machine that is connected locally on one network to talk
-to another machine that is also connected to that same network (as opposed to a
-machine that wants to communicate over multiple networks).
-
-We then, after seeing the ARP packets go back and forth (which establish the
-ability for those two containers to talk to each other on the local network), we
-see the ICMP echo-request and echo-reply packets go back and forth.
+Basically, all you need to know about this is that `ping` is a program that sends packets across the network using a protocol called ICMP, which stands for Internet Control Message Protocol. `echo request` and `echo reply` are two types of ICMP message. You can read more about them [here](https://docs.netapp.com/us-en/e-series-santricity/sm-hardware/what-are-icmp-ping-responses.html) if you want to know more. What we see here in this ping message is that it has both sent a packet to the destination (echo request), and the destination has replied (echo reply). The `icmp_seq=3` designation marks each individual request/response pair. If the ping did not go through, you might see various error messages, but the most common is that the `ping` command replies with `Request timeout for icmp_seq 0` type messages.
+
+On the container that is being pinged (pippin), we see quite a bit more information from tcpdump. First, we see a series of messages that have `ARP` in them. ARP, which stands for Address Resolution Protoco,  is a protocol that allows a machine that is connected locally on one network to talk to another machine that is also connected to that same network (as opposed to a machine that wants to communicate over multiple networks). You can learn all about ARP [here](https://www.fortinet.com/resources/cyberglossary/what-is-arp) if you are interested in diving deeper.
```

Comment:
Oh blergh. We cover this in part in the [IP and MAC address appendix](https://github.com/psbanka/build-your-own-internet/blob/004-ip-and-ethernet/chapters/appendix/ip-and-mac-addresses.md)... Maybe in addition to sending people out to the internet we could say that we'll discuss this a bit more in future chapters? So, like 

> We'll talk about this crucial protocol more over the coming chapters, but you can get a head start learning about ARP [here](https://www.fortinet.com/resources/cyberglossary/what-is-arp).